query-engine/core/src/executor/execute_operation.rs

```diff
@@ -44,16 +44,25 @@ pub async fn execute_many_operations(
 
     let mut results = Vec::with_capacity(queries.len());
 
-    for (query_graph, serializer) in queries {
+    for (i, (query_graph, serializer)) in queries.into_iter().enumerate() {
         increment_counter!(PRISMA_CLIENT_QUERIES_TOTAL);
         let operation_timer = Instant::now();
         let interpreter = QueryInterpreter::new(conn);
         let result = QueryPipeline::new(query_graph, interpreter, serializer)
             .execute(trace_id.clone())
-            .await?;
+            .await;
 
         histogram!(PRISMA_CLIENT_QUERIES_HISTOGRAM_MS, operation_timer.elapsed());
```

Comment:
This might be an unintentional change, but previously this metric wouldn't have been logged if there was an error, right? The new behaviour sounds more correct to me, but we might want to check what happens in other places and if it's consistent.

Reply:
Yep, in `execute_single_query` we write this histogram regardless of whether the result is `Ok` or `Err`. Looks like another unrelated bug fixed here at the same time 😄

Reply:
good catch.